### PR TITLE
Qt recipes now use CPU_COUNT

### PR DESCRIPTION
**Pull request by gbaty**
Qt recipes now use CPU_COUNT variable everywhere instead of "sysctl -n hw.ncpu"

### CODE REVIEW

File: qt5/build.sh [hunk 1]
```diff
@@ -20,8 +20,6 @@ if [ `uname` == Darwin ]; then
     CONFIG_OPTS+=" -platform macx-g++ -no-framework -no-c++11"
     CONFIG_OPTS+=" -no-mtdev -no-harfbuzz -no-xinput2 -no-xcb-xlib"
     CONFIG_OPTS+=" -no-libudev -no-egl"
-
-    MAKE_JOBS=$(sysctl -n hw.ncpu)
```
Comment (ccordoba12) [inline qt5/build.sh]: Please leave this one. It's more reliable than `CPU_COUNT` in Mac.

File: qt/build.sh [hunk 2]
```diff
@@ -53,7 +54,7 @@ if [ `uname` == Darwin ]; then
         -opensource -verbose -openssl -no-framework -system-libpng \
         -arch `uname -m` -L $PREFIX/lib -I $PREFIX/include
 
-    make -j $(sysctl -n hw.ncpu)
```
Comment (ccordoba12) [inline qt/build.sh]: Again, please leave things as they are on Mac :-)

**Comment by gbaty** [comment]
Thanks for comment.
The official documentation is a bit confusing, as CPU_COUNT is defined in [Environment Variables](http://conda.pydata.org/docs/building/environment-vars.html) and not in subsection "On Linux, we have:". So It seems to be the right way of setting number of CPU. That is why I have written this pr.
Do I write an issue for that documentation point ? A fix in [conda-build/environ.py:101](https://github.com/conda/conda-build/blob/master/conda_build/environ.py#L101) or in documentation ? Or do you think it must be fixed in multiprocessing.cpu_count function ?

Thank you

**Comment by ccordoba12** [comment]
It's just more reliable on Mac to use `sysctl -n hw.ncpu`.

I think it's better for you to open a new pull request against `conda-build`, using `sysctl -n hw.ncpu` instead of `multiprocessing.cpu_count` in Mac to set `CPU_COUNT`.

When that's done, we can merge this pull request.

**Comment by gbaty** [comment]
Thanks for your answer. It will be fixed in a pull-request in conda-build repository.

**Comment by msarahan** [comment]
Can this PR be closed? Is the conda build PR the better solution here?

**Comment by ccordoba12** [comment]
The conda-build PR needs to be merged first before this PR can be applied.

**Pull request closed without merging**